### PR TITLE
docs -  ec2_vpn_gateway

### PR DESCRIPTION
**Pull request by catlike**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_ec2_vpn_gateway/ec2_vpn_gateway.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+resource "awscc_ec2_vpn_gateway" "example_vpn_gateway" {
+  type = "ipsec.1"
+  tags = [{
+    key   = "Name"
+    value = "Example VPN Gateway"
+  }]
```
Comment (jackywong-amazon) [inline examples/resources/awscc_ec2_vpn_gateway/ec2_vpn_gateway.tf:6]: probably add this 

tags = [{
    key   = "Modified By"
    value = "AWSCC"
  }]

File: templates/resources/ec2_vpn_gateway.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Example Usage
+
+### First example - create basic VPN gateway
+
+{{ tffile (printf "examples/resources/%s/ec2_vpn_gateway.tf" .Name)}}
+
+### Second example - create VPN specifying the private Autonomous System Number (ASN) for the Amazon side of a BGP session.
+Description about the second example
+{{ tffile (printf "examples/resources/%s/ec2_vpn_gateway_asn.tf" .Name)}}
```
Comment (jackywong-amazon) [inline templates/resources/ec2_vpn_gateway.md.tmpl:20]: did you want to have another space here?

**Comment by catlike** [comment]
@jackywong-amazon thank you for the feedback and KEEN eye.

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks !

**Comment by marcosentino** [comment]
@catlike Thank you for the contribution!

**Pull request merged**